app/controls/form/form.js

```diff
@@ -165,6 +165,15 @@ define([
             var wrapper = views[ type + 'Wrapper' ] || views.wrapper,
                 options;
 
+            var elementsToStringify = ['placeholder'];
+            var elementData = $el.data();
+
+            $.each(elementData, function(key, val) {
+               if(typeof val === 'number' && elementsToStringify.indexOf(key) > -1) {
```

Comment:
indexOf is IE9 +, also remember that $ is not meant to be global here.
I would also reverse this, looping over the array first saves the looping that wont ever happen.
Underscore is in scope at _ (_.forEach)
I can write up an example, but I think you'd prefer to work it out yourself ;)